pytensor/configparser.py

```diff
@@ -175,7 +165,7 @@ class PyTensorConfigParser:
     # add_blas_configvars
     blas__ldflags: str
     blas__check_openmp: bool
-    # add CUDA (?)
+    # add CUDA
     cuda__root: Path | None
     dnn__base_path: Path | None
     dnn__library_path: Path | None
```

Comment:
Are these used anywhere? We don't have CUDA/GPU stuff anymore

Reply:
No indeed, only in `configdefaults.py` but this can be removed too, thanks